.coderabbit.yaml

```
@@ -0,0 +1,72 @@
+    - ".next/**"
+    - "coverage/**"
+    - "*.lock"
+
+# Language-specific settings
+language_settings:
+  typescript:
+    # Check for TypeScript best practices
+    strict_mode: true
+    # Suggest modern syntax
+    modern_syntax: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)